cypress/support/fragments/invoices/invoices.js

```diff
@@ -120,9 +120,10 @@ export default {
       })
       .then(({ body }) => body);
   },
-  createInvoiceViaApi({ vendorId, accountingCode, exportToAccounting }) {
+  createInvoiceViaApi({ vendorId, accountingCode, fiscalYearId, exportToAccounting }) {
```

Comment:
It's not a necessary element. You don't need to add it. You can create an invoice without it.

Reply:
I need this for test purposes. Test precondition contain requirement to create Invoice with specified FY

Reply:
So it's only in your test. Not all tests are like this. Why change the whole fragment?

Reply:
Other tests will not be affected, if 'fiscalYearId' is not provided as an argument then request body to create invoice will not contain 'fiscalYearId' property. Previous behavior of the function will not change.